src/components/AppLayout/index.tsx

```diff
@@ -740,6 +741,12 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 				displayName={mainDisplay}
 				isVerified={isGood && !isIdentityUnverified}
 			/>
+			{id && isIdentityUnverified && network === 'polkadot' && (
+				<UnverifiedUserNudge
+					handleSetIdentityClick={handleIdentityButtonClick}
+					isIdentitySet={email ? true : false}
```

Comment:
this is wrong handling @adetoye-dev

Reply:
Alright, can you help with more context  on the changes required 🙏

Thanks!

Reply:
Updated.
Thanks, Kanishka!